src/lucky/exceptions.cr

```diff
@@ -12,5 +12,23 @@ module Lucky
 
     class InvalidMessage < Base
     end
+
+    class InvalidParam < Base
+      def initialize(@param_name : String, @param_value : String, @param_type : String)
```

Comment:
Could you add getters for all these? That way people can use them in custom error message if they want to